src/lib/pay-request/services/admin_users/types.ts

```diff
@@ -61,7 +61,23 @@ export interface ServiceRole {
   role: Role;
 }
 
-export interface User {
+export class User {
+  constructor (userData: any) {
+    if (!userData) {
+      throw Error('Must provide data')
+    }
+    this.external_id = userData.external_id
+    this.email = userData.email || ''
+    this.service_roles = userData.service_roles
+    this.otp_key = userData.otp_key || ''
+    this.telephone_number = userData.telephone_number || ''
+    this.disabled = userData.disabled ? userData.disabled : false
+    this.session_version = userData.session_version || 0
+    this.features = (userData.features || '').split(',').map((feature: string) => feature.trim())
```

Comment:
splitting an empty string yields `['']` so you may be better off doing `userData.features ? userData.features.split(...).map(...) : []`